runtime/src/weights/mod.rs

```diff
@@ -9,4 +9,4 @@ pub mod pallet_encointer_democracy;
 pub mod pallet_encointer_faucet;
 pub mod pallet_encointer_reputation_commitments;
 pub mod pallet_encointer_scheduler;
-pub mod pallet_treasury;
+// pub mod pallet_treasury;
```

Comment:
Intended?

Reply:
yes. it causes warnings and we're not actually using it right now. will reconsider when we benchamrk the next time